ARM.CMSIS.pdsc

```diff
@@ -668,6 +682,24 @@
       </files>
     </component>
 
+    <component Cclass="CMSIS" Cgroup="CORE" Cversion="1.0.0"  condition="ARMv8-A Device" >
```

Comment:
This component has identical content as for ARMv7-A Device. Can't we just put them all in one by combining the condition? Technically, we merged all files back into a single core folder. The only difference between M and A class is the documentation reference.

Reply:
changed it to the Cortex-M way